lms/djangoapps/verify_student/views.py

```diff
@@ -1105,7 +1105,7 @@ def _send_confirmation_email(self, user):
 
         subject = _("Verification photos received")
         message = render_to_string('emails/photo_submission_confirmation.txt', context)
-        from_address = configuration_helpers.get_value('default_from_email', settings.DEFAULT_FROM_EMAIL)
+        from_address = configuration_helpers.get_value('email_from_address', settings.DEFAULT_FROM_EMAIL)
```

Comment:
@douglashall we should not. Currently when you want to set an override for settings.DEFAULT_FROM_EMAIL in a white label, you already use `email_from_address`, so this change should correct the situation where you set `email_from_address` for a white label and still get emails with the wrong sender address. This is how we noticed this difference at all.